deploy/restartComponent.sh

```diff
@@ -8,7 +8,14 @@
 
 HOST=$(hostname)
 DATENOW=$(date +%s)
-DEST_NAME=cms-wmcore-team
+# look-up alert emails from WMA secret file
+EMAIL_ALERT_WM=`grep EMAIL_ALERT_WM $WMA_SECRETS_FILE`
+EMAIL_ALERT_T0=`grep EMAIL_ALERT_T0 $WMA_SECRETS_FILE`
+EMAIL_ALERT_PNR=`grep EMAIL_ALERT_PNR $WMA_SECRETS_FILE`
+
+[[ -z $EMAIL_ALERT_WM ]] && {echo "ERROR: unable to find EMAIL_ALERT_WM in $WMA_SECRETS_FILE"; exit 1;}
+[[ -z $EMAIL_ALERT_T0 ]] && {echo "ERROR: unable to find EMAIL_ALERT_T0 in $WMA_SECRETS_FILE"; exit 1;}
```

Comment:
And if you decide to stick with the multiple variables, this code needs to be revisited, otherwise it would make a requirement that we need to define all 3 variables.